core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -130,8 +130,6 @@
     public static final String ENABLE_COORDINATOR_DYNAMIC_FILTERS_DISTRIBUTION = "enable_coordinator_dynamic_filters_distribution";
     public static final String ENABLE_LARGE_DYNAMIC_FILTERS = "enable_large_dynamic_filters";
     public static final String QUERY_MAX_MEMORY_PER_NODE = "query_max_memory_per_node";
-    public static final String QUERY_MAX_TOTAL_MEMORY_PER_NODE = "query_max_total_memory_per_node";
-    public static final String QUERY_MAX_TOTAL_MEMORY_PER_TASK = "query_max_total_memory_per_task";
```

Comment:
where did it go? Maybe we do not need a session property for it. But then it would be better to have removal in separate commit.

Reply:
It was used like this:
https://github.com/trinodb/trino/blob/c88d96811eb8720b8bdb9e4d75cb152ff0c6e352/core/trino-main/src/main/java/io/trino/execution/SqlTaskManager.java#L415-L426

i.e. it was used to override task _system_ memory only, which didn't look very practical.
I can restore, but please advice what logic you want to see there.

Reply:
ok, restored.

Reply:
It was used for total task memory. There was no separate check for system and total for per-task limits. (at least that was an intention)

> ok, restored.

Thx. If sth is wrong with logic here lets handle that as a followup.